src/PhpSpreadsheet/Writer/Xlsx.php

```diff
@@ -232,6 +232,7 @@ public function save($pFilename)
                         //signed macros ?
                         // Yes : add the certificate file and the related rels file
                         $zip->addFromString('xl/vbaProjectSignature.bin', $this->spreadSheet->getMacrosCertificate());
+                        // @scrutinizer ignore-call
```

Comment:
A better way to fix this is to introduced dedicated properties for each parts, instead of using a loosely type array. So this should be reverted, and introducing properties for each part should be another dedicated PR.